package demo3;

public class RuntimeExceptionDemo6 {

	public static void main(String[] args) {
		
//		ArithmeticException	:	0으로 어떤 수를 나눌 때 발생한다.
		
		System.out.println(3/0);
	}
}
